.github/workflows/reusable-satellite-smoke-test.yml

```
@@ -0,0 +1,71 @@
+name: Satellite Smoke Test
+
+on:
+  workflow_call:
+    inputs:
+      BUILT_EARTHLY_PATH:
+        required: true
+        type: string
+      RUNS_ON:
+        required: true
+        type: string
+      SUDO:
+        type: string
+        required: false
+      BINARY:
+        required: true
+        type: string
+      SATELLITE_NAME:
+        required: true
+        type: string
+      EARTHLY_ORG:
+        required: false
+        type: string
+      USE_VPN:
+        required: false
+        type: boolean
+
+jobs:
+  satellite-smoke-test:
+    runs-on: ${{inputs.RUNS_ON}}
+    env:
+      FORCE_COLOR: 1
+      EARTHLY_TOKEN: "${{ secrets.EARTHLY_TOKEN }}"
+      EARTHLY_ORG: "${{inputs.EARTHLY_ORG}}"
```

nit: to be consistent
```suggestion
      EARTHLY_ORG: "${{ inputs.EARTHLY_ORG }}"
```